sematic/examples/bert_fine_tune_scam/requirements.txt

```diff
@@ -0,0 +1,8 @@
+# This is where you list the third-party dependencies of your pipeline
+
+# Examples:
+# pandas
+# seaborn
+# torch
```

Comment:
Please remove comments.

Also this probably needs `numpy`, `plotly`, `torch`, and `pandas`.